src/provider/formattingProvider.ts

```diff
@@ -87,16 +88,11 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
                 }
             }
 
-            if (purityText.match(/}/) != null) {
-                let temp = purityText.match(/}/).length;
-                const t2 = purityText.match(/{[^{}]*}/);
-                if (t2) {
-                    temp = temp - t2.length;
-                }
-                deep -= temp;
-                if (temp > 0) {
-                    notDeep = false;
+            if (purityText.includes("}")) {
+                if (!purityText.includes("}:")) {
+                    deep--;
                 }
+                notDeep = false;
             }
 
             if (oneCommandCode && purityText.match(/{/) != null) {
```

Comment:
If you'd like, you can remove this use of regex: `... && purityText.includes('{')) {`